internal/validation/strings.go

```diff
@@ -12,5 +14,15 @@ var (
 	StringIsPortDefinition = validation.StringMatch(
 		regexp.MustCompile("^(([0-9]+(,[0-9]+){0,9})|([0-9]+-[0-9]+)|(any))$"),
 		"port definition has to be: up to 10 comma sepparated numbers (22,23), range (20-23) or word 'any'")
-	StringIsSpeedBand = validation.StringMatch(regexp.MustCompile("^[0-9]+(MB|GB)$"), "SpeedBand should consist of digit followed by MB or GB")
+	StringIsSpeedBand   = validation.StringMatch(regexp.MustCompile("^[0-9]+(MB|GB)$"), "SpeedBand should consist of digit followed by MB or GB")
+	StringIsCountryCode = stringvalidator.RegexMatches(StringToRegex("(?i)^[a-z]{2}$"), "Address country must be a two letter code (ISO 3166-1 alpha-2)")
```

Comment:
Would `validation.Stringmatch(regexp.MustCompile(...` work here as it does on the line above?  I'm not clear what we gain by using `stringvalidator.RegexMatches`, especially if that requires adding a helper function that silently swallows a potential error.

Reply:
Note this still needs to be addressed.  A direct call to `regexp.MustCompile` will panic if the regex is invalid.  The `StringToRegex` helper will log an error and return nil and it's not clear what would happen in that case.  We should remove the helper.